Traceback (most recent call last):
  File "C:\Python310\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\matth\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\matth\AppData\Roaming\Python\Python310\site-packages\jupyter_core\utils\__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
  File "C:\Python310\lib\asyncio\base_events.py", line 649, in run_until_complete
    return future.result()
  File "C:\Users\matth\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "C:\Users\matth\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\matth\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Load a pretrained YOLOv8n model
model = YOLO('../runs/pose/train3/weights/best.pt')
------------------

----- stdout -----
⚠️ GitHub assets check failure for https://api.github.com/repos/ultralytics/assets/releases/tags/v8.1.0: 403 rate limit exceeded
----- stdout -----
⚠️ GitHub assets check failure for https://api.github.com/repos/ultralytics/assets/releases/latest: 403 rate limit exceeded
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mFileNotFoundError[0m                         Traceback (most recent call last)
Cell [1;32mIn[5], line 2[0m
[0;32m      1[0m [38;5;66;03m# Load a pretrained YOLOv8n model[39;00m
[1;32m----> 2[0m model [38;5;241m=[39m [43mYOLO[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43m../runs/pose/train3/weights/best.pt[39;49m[38;5;124;43m'[39;49m[43m)[49m

File [1;32mC:\Python310\lib\site-packages\ultralytics\models\yolo\model.py:23[0m, in [0;36mYOLO.__init__[1;34m(self, model, task, verbose)[0m
[0;32m     20[0m     [38;5;28mself[39m[38;5;241m.[39m[38;5;18m__dict__[39m [38;5;241m=[39m new_instance[38;5;241m.[39m[38;5;18m__dict__[39m
[0;32m     21[0m [38;5;28;01melse[39;00m:
[0;32m     22[0m     [38;5;66;03m# Continue with default YOLO initialization[39;00m
[1;32m---> 23[0m     [38;5;28;43msuper[39;49m[43m([49m[43m)[49m[38;5;241;43m.[39;49m[38;5;21;43m__init__[39;49m[43m([49m[43mmodel[49m[38;5;241;43m=[39;49m[43mmodel[49m[43m,[49m[43m [49m[43mtask[49m[38;5;241;43m=[39;49m[43mtask[49m[43m,[49m[43m [49m[43mverbose[49m[38;5;241;43m=[39;49m[43mverbose[49m[43m)[49m

File [1;32mC:\Python310\lib\site-packages\ultralytics\engine\model.py:134[0m, in [0;36mModel.__init__[1;34m(self, model, task, verbose)[0m
[0;32m    132[0m     [38;5;28mself[39m[38;5;241m.[39m_new(model, task[38;5;241m=[39mtask, verbose[38;5;241m=[39mverbose)
[0;32m    133[0m [38;5;28;01melse[39;00m:
[1;32m--> 134[0m     [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_load[49m[43m([49m[43mmodel[49m[43m,[49m[43m [49m[43mtask[49m[38;5;241;43m=[39;49m[43mtask[49m[43m)[49m
[0;32m    136[0m [38;5;28mself[39m[38;5;241m.[39mmodel_name [38;5;241m=[39m model

File [1;32mC:\Python310\lib\site-packages\ultralytics\engine\model.py:215[0m, in [0;36mModel._load[1;34m(self, weights, task)[0m
[0;32m    213[0m suffix [38;5;241m=[39m Path(weights)[38;5;241m.[39msuffix
[0;32m    214[0m [38;5;28;01mif[39;00m suffix [38;5;241m==[39m [38;5;124m"[39m[38;5;124m.pt[39m[38;5;124m"[39m:
[1;32m--> 215[0m     [38;5;28mself[39m[38;5;241m.[39mmodel, [38;5;28mself[39m[38;5;241m.[39mckpt [38;5;241m=[39m [43mattempt_load_one_weight[49m[43m([49m[43mweights[49m[43m)[49m
[0;32m    216[0m     [38;5;28mself[39m[38;5;241m.[39mtask [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mmodel[38;5;241m.[39margs[[38;5;124m"[39m[38;5;124mtask[39m[38;5;124m"[39m]
[0;32m    217[0m     [38;5;28mself[39m[38;5;241m.[39moverrides [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mmodel[38;5;241m.[39margs [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39m_reset_ckpt_args([38;5;28mself[39m[38;5;241m.[39mmodel[38;5;241m.[39margs)

File [1;32mC:\Python310\lib\site-packages\ultralytics\nn\tasks.py:780[0m, in [0;36mattempt_load_one_weight[1;34m(weight, device, inplace, fuse)[0m
[0;32m    778[0m [38;5;28;01mdef[39;00m [38;5;21mattempt_load_one_weight[39m(weight, device[38;5;241m=[39m[38;5;28;01mNone[39;00m, inplace[38;5;241m=[39m[38;5;28;01mTrue[39;00m, fuse[38;5;241m=[39m[38;5;28;01mFalse[39;00m):
[0;32m    779[0m [38;5;250m    [39m[38;5;124;03m"""Loads a single model weights."""[39;00m
[1;32m--> 780[0m     ckpt, weight [38;5;241m=[39m [43mtorch_safe_load[49m[43m([49m[43mweight[49m[43m)[49m  [38;5;66;03m# load ckpt[39;00m
[0;32m    781[0m     args [38;5;241m=[39m {[38;5;241m*[39m[38;5;241m*[39mDEFAULT_CFG_DICT, [38;5;241m*[39m[38;5;241m*[39m(ckpt[38;5;241m.[39mget([38;5;124m"[39m[38;5;124mtrain_args[39m[38;5;124m"[39m, {}))}  [38;5;66;03m# combine model and default args, preferring model args[39;00m
[0;32m    782[0m     model [38;5;241m=[39m (ckpt[38;5;241m.[39mget([38;5;124m"[39m[38;5;124mema[39m[38;5;124m"[39m) [38;5;129;01mor[39;00m ckpt[[38;5;124m"[39m[38;5;124mmodel[39m[38;5;124m"[39m])[38;5;241m.[39mto(device)[38;5;241m.[39mfloat()  [38;5;66;03m# FP32 model[39;00m

File [1;32mC:\Python310\lib\site-packages\ultralytics\nn\tasks.py:708[0m, in [0;36mtorch_safe_load[1;34m(weight)[0m
[0;32m    700[0m [38;5;28;01mtry[39;00m:
[0;32m    701[0m     [38;5;28;01mwith[39;00m temporary_modules(
[0;32m    702[0m         {
[0;32m    703[0m             [38;5;124m"[39m[38;5;124multralytics.yolo.utils[39m[38;5;124m"[39m: [38;5;124m"[39m[38;5;124multralytics.utils[39m[38;5;124m"[39m,
[1;32m   (...)[0m
[0;32m    706[0m         }
[0;32m    707[0m     ):  [38;5;66;03m# for legacy 8.0 Classify and Pose models[39;00m
[1;32m--> 708[0m         ckpt [38;5;241m=[39m [43mtorch[49m[38;5;241;43m.[39;49m[43mload[49m[43m([49m[43mfile[49m[43m,[49m[43m [49m[43mmap_location[49m[38;5;241;43m=[39;49m[38;5;124;43m"[39;49m[38;5;124;43mcpu[39;49m[38;5;124;43m"[39;49m[43m)[49m
[0;32m    710[0m [38;5;28;01mexcept[39;00m [38;5;167;01mModuleNotFoundError[39;00m [38;5;28;01mas[39;00m e:  [38;5;66;03m# e.name is missing module name[39;00m
[0;32m    711[0m     [38;5;28;01mif[39;00m e[38;5;241m.[39mname [38;5;241m==[39m [38;5;124m"[39m[38;5;124mmodels[39m[38;5;124m"[39m:

File [1;32mC:\Python310\lib\site-packages\torch\serialization.py:986[0m, in [0;36mload[1;34m(f, map_location, pickle_module, weights_only, mmap, **pickle_load_args)[0m
[0;32m    983[0m [38;5;28;01mif[39;00m [38;5;124m'[39m[38;5;124mencoding[39m[38;5;124m'[39m [38;5;129;01mnot[39;00m [38;5;129;01min[39;00m pickle_load_args[38;5;241m.[39mkeys():
[0;32m    984[0m     pickle_load_args[[38;5;124m'[39m[38;5;124mencoding[39m[38;5;124m'[39m] [38;5;241m=[39m [38;5;124m'[39m[38;5;124mutf-8[39m[38;5;124m'[39m
[1;32m--> 986[0m [38;5;28;01mwith[39;00m [43m_open_file_like[49m[43m([49m[43mf[49m[43m,[49m[43m [49m[38;5;124;43m'[39;49m[38;5;124;43mrb[39;49m[38;5;124;43m'[39;49m[43m)[49m [38;5;28;01mas[39;00m opened_file:
[0;32m    987[0m     [38;5;28;01mif[39;00m _is_zipfile(opened_file):
[0;32m    988[0m         [38;5;66;03m# The zipfile reader is going to advance the current file position.[39;00m
[0;32m    989[0m         [38;5;66;03m# If we want to actually tail call to torch.jit.load, we need to[39;00m
[0;32m    990[0m         [38;5;66;03m# reset back to the original position.[39;00m
[0;32m    991[0m         orig_position [38;5;241m=[39m opened_file[38;5;241m.[39mtell()

File [1;32mC:\Python310\lib\site-packages\torch\serialization.py:435[0m, in [0;36m_open_file_like[1;34m(name_or_buffer, mode)[0m
[0;32m    433[0m [38;5;28;01mdef[39;00m [38;5;21m_open_file_like[39m(name_or_buffer, mode):
[0;32m    434[0m     [38;5;28;01mif[39;00m _is_path(name_or_buffer):
[1;32m--> 435[0m         [38;5;28;01mreturn[39;00m [43m_open_file[49m[43m([49m[43mname_or_buffer[49m[43m,[49m[43m [49m[43mmode[49m[43m)[49m
[0;32m    436[0m     [38;5;28;01melse[39;00m:
[0;32m    437[0m         [38;5;28;01mif[39;00m [38;5;124m'[39m[38;5;124mw[39m[38;5;124m'[39m [38;5;129;01min[39;00m mode:

File [1;32mC:\Python310\lib\site-packages\torch\serialization.py:416[0m, in [0;36m_open_file.__init__[1;34m(self, name, mode)[0m
[0;32m    415[0m [38;5;28;01mdef[39;00m [38;5;21m__init__[39m([38;5;28mself[39m, name, mode):
[1;32m--> 416[0m     [38;5;28msuper[39m()[38;5;241m.[39m[38;5;21m__init__[39m([38;5;28;43mopen[39;49m[43m([49m[43mname[49m[43m,[49m[43m [49m[43mmode[49m[43m)[49m)

[1;31mFileNotFoundError[0m: [Errno 2] No such file or directory: '..\\runs\\pose\\train3\\weights\\best.pt'

